src/SCRIPTS/BF/X7/x7pre.lua

```diff
@@ -8,7 +8,8 @@ PageFiles =
     "filters1.lua",
     "filters2.lua",
     "pwm.lua",
-    "vtx.lua"
+    "vtx.lua",
+    "esc.lua",
```

Comment:
Shouldn't this go into `X9/` and `HORUS/` as well?

Reply:
Sure, once you are happy with my approach I'll roll it out to the other radios too.